modules/api/src/main/java/org/apache/fluo/api/config/FluoConfiguration.java

```diff
@@ -275,12 +275,27 @@ public void validate() {
     getZookeeperTimeout();
   }
 
+  /**
+   * Sets the {@value #CONNECTION_APPLICATION_NAME_PROP}
+   * @param applicationName
+   * @return
+   */
   public FluoConfiguration setApplicationName(String applicationName) {
     verifyApplicationName(applicationName);
     setProperty(CONNECTION_APPLICATION_NAME_PROP, applicationName);
     return this;
   }
-
+  /**
+   * Returns the application name after verification to avoid characters Zookeeper does not like
+   * in nodes and Hadoop does not like in HDFS paths.
+<<<<<<< HEAD
```

Comment:
This is created by git when there is a merge conflict.  Did you try to merge some other branch?

When you see this just need to look at the content above and below the `=======` and manually merge it.   Then get rid of the `<<<<<<< HEAD` , `=======`, and `>>>>>>> fb07d624c1d975c397e2dd257ce4e646221ec5b6`